builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -330,6 +313,50 @@ node() {
   }
 }
 
+def handleS3BucketError(deployOutput, s3BucketName, environment_logical_id, envBucketKey, isEventSchdld) {
+  def s3BucketErrString = "CloudFormation - CREATE_FAILED - AWS::S3::Bucket - "
+  if (deployOutput.contains(s3BucketErrString)) {
+    if (lambdaEvents.checkS3BucketExists(s3BucketName)) {
+      isLambdaUpdateRequired = true
+      lambdaEvents.removeS3EventsFromServerless(isEventSchdld)
+      echoServerlessFile()
+      sh "serverless remove --stage $environment_logical_id -v --bucket ${configLoader.AWS.S3[envBucketKey]}"
+      def redeployOutput = servelessDeploy(environment_logical_id, envBucketKey)
+      if (redeployOutput != 'success') {
+        echo "Exception occured while serverless deployment to ${environment_logical_id} environment : $redeployOutput"
+        error "Exception occured while serverless deployment to ${environment_logical_id} environment"
+      }
+    } else {
+      error "Error occured while accessing the s3 bucket"
+    }
+  } else {
+    echo "Exception occured while serverless deployment to ${environment_logical_id} environment : $deployOutput"
+    error "Exception occured while serverless deployment to ${environment_logical_id} environment"
+  }
+}
+
+def handleSqsError(deployOutput, event_source_sqs_arn, environment_logical_id, envBucketKey) {
+  def sqsErrString = "CloudFormation - CREATE_FAILED - AWS::SQS::Queue - "
+  if (deployOutput.contains(sqsErrString)) {
+    def event_source_sqs = event_source_sqs_arn.split(":(?!.*:.*)")[1]
+    if (lambdaEvents.checkSqsQueueExists(event_source_sqs)) {
+      lambdaEvents.updateSqsResourceServerless()
```

Comment:
This code should work when sqs queue exists. So why are you adding the queue here. i am confused